tools/enumerate-devices/rs-enumerate-devices.cpp

```diff
@@ -180,12 +182,14 @@ int main(int argc, char** argv) try
 {
     CmdLine cmd("librealsense rs-enumerate-devices tool", ' ', RS2_API_VERSION_STR);
 
-    SwitchArg compact_view_arg("s", "short", "Provide short summary of the devices");
-    SwitchArg show_options_arg("o", "option", "Show all supported options per subdevice");
+    SwitchArg short_view_arg("s", "short", "Provide a one-line summary of the devices");
+    SwitchArg compact_view_arg("S", "compact", "Provide a short summary of the devices");
```

Comment:
You're changing the option for compact from -s to -S? That changes existing usage... intended?

Reply:
Correct. The original `-s` meant to be a one-liner, as noted in #5423